Webpack build missing dependency bcrypto/compat

**Version**
bcoin v1.0.2

**Expected**

Webpack builds to work

**Actual**

After getting past issues with installing webpack globally, there is an error generated due to `./compat` path no-longer existing in bcrypto.
```
$ npm run webpack

> bcoin@1.0.2 webpack /home/user/bcoin
> webpack --mode production --config webpack.browser.js

✖ ｢config｣: An error occurred while trying to load /home/bgf/code/bcoin-org/bcoin/webpack.browser.js
              Did you forget to specify a --require?
/home/bgf/code/bcoin-org/bcoin/node_modules/@webpack-contrib/config-loader/lib/load.js:85
    throw new LoadConfigError(e, configPath);
    ^

LoadConfigError: Cannot find module 'bcrypto/compat'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/user/bcoin/webpack.browser.js:24:12)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at loadJs (/home/user/bcoin/node_modules/cosmiconfig/dist/loaders.js:8:18)
    at Explorer.loadFileContent (/home/user/bcoin/node_modules/cosmiconfig/dist/createExplorer.js:226:12)
    at Explorer.createCosmiconfigResultSync (/home/user/bcoin/node_modules/cosmiconfig/dist/createExplorer.js:259:31)
    at cacheWrapper (/home/user/bcoin/node_modules/cosmiconfig/dist/createExplorer.js:288:27)
```

**Reproduce**
```
npm run webpack
```

--- a/webpack.app.js
+++ b/webpack.app.js
@@ -18,8 +18,7 @@ module.exports = {
   },
   resolve: {
     modules: ['node_modules'],
-    extensions: ['-browser.js', '.js', '.json'],
-    alias: require('bcrypto/compat')
+    extensions: ['-browser.js', '.js', '.json']
   },
   plugins: [
     new webpack.DefinePlugin({

--- a/webpack.browser.js
+++ b/webpack.browser.js
@@ -20,8 +20,7 @@ module.exports = {
   },
   resolve: {
     modules: ['node_modules'],
-    extensions: ['-browser.js', '.js', '.json'],
-    alias: require('bcrypto/compat')
+    extensions: ['-browser.js', '.js', '.json']
   },
   plugins: [
     new webpack.DefinePlugin({

